### PR TITLE
python3Packages.beautiful-date: init at 2.2.0

### PR DESCRIPTION
**Pull request by mbalatsko**
## Description of changes
Add python package beautiful-date in version 2.2.0 to default python3 packages.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mbalatsko** [comment]
Output of nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD" built on x86_64-linux
```
2 packages built:
python310Packages.beautiful-date python311Packages.beautiful-date
```

File: pkgs/development/python-modules/beautiful-date/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "beautiful_date" ];
+
+  meta = with lib; {
+    description = "Simple and beautiful way to create date and datetime objects in Python";
+    homepage = "https://github.com/kuzmoyev/beautiful-date";
+    license = licenses.mit;
+    maintainers = with maintainers; [ ];
```
Comment (fabaff) [inline pkgs/development/python-modules/beautiful-date/default.nix:37]: Set yourself as maintainer :wink:

File: pkgs/development/python-modules/beautiful-date/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-5xRmHaAPf1ps75cOINHkHT1aYb5UGLZGl0OHVQaMES0=";
+  };
+
+  propagatedBuildInputs = [
+    python-dateutil
+  ];
+
+  checkInputs = [
```
Comment (natsukium) [inline pkgs/development/python-modules/beautiful-date/default.nix:27]:
```suggestion
  nativeCheckInputs = [
```
Please use nativeCheckInputs instead.

**Comment by natsukium** [comment]
@ofborg build python310Packages.beautiful-date

**Review by natsukium — APPROVED** [review]
LGTM

**Pull request merged**